src/validateData.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable max-len */
+function validateData(text, toCase) {
+  const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errorMessages = {
+    emptyTextMessage: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    emptyCaseMessage: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    invalidCaseMessage: `This case is not supported. Available cases: ${validCases.join(', ')}.`,
+  };
```

Comment:
Create these variables outside the function to avoid re-creating whenever it is executed